src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/Microsoft.VisualStudio.LanguageServices.Razor.Test.csproj

```diff
@@ -24,6 +24,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\src\Microsoft.CodeAnalysis.Remote.Razor\Microsoft.CodeAnalysis.Remote.Razor.csproj" />
```

Comment:
I wonder if we'll be breaking out another test library later, since `MS.CA.Remote.Razor` should never be loaded in Visual Studio.

Reply:
I considered doing this as a separate project now, but it didn't seem to add much value. Certainly in future we'll need to as the cohosting bits will have to move somewhere other than the VS layer, so they can work in VS Code. Once that happens we'll definitely need something new.